command/meta.go

```diff
@@ -112,21 +112,30 @@ func (m *ApiMeta) Process(args []string) ([]string, error) {
 	}
 
 	// CLI configuration.
+	userHome := ""
 	usr, err := user.Current()
 	if err != nil {
-		m.Ui.Error(fmt.Sprintf("Could not read current user from environment, failing."))
-		return args, err
+		// Fallback by trying to read $HOME
+		userHome = os.Getenv("HOME")
+		if userHome != "" {
+			err = nil
+		} else {
+			m.Ui.Error(fmt.Sprintf("Could not read current user from environment, failing."))
+			return args, err
+		}
+	} else {
+		userHome = usr.HomeDir
 	}
 
 	// TODO(jacobkiefer): Add flag for config location?
-	m.configLocation = filepath.Join(usr.HomeDir, ".spin", "config")
+	m.configLocation = filepath.Join(userHome, ".spin", "config")
 	yamlFile, err := ioutil.ReadFile(m.configLocation)
 	if err != nil {
 		m.Ui.Warn(fmt.Sprintf("Could not read configuration file from %s.", m.configLocation))
 	}
 
 	if yamlFile != nil {
-		err = yaml.UnmarshalStrict(yamlFile, &m.Config)
+		err = yaml.Unmarshal(yamlFile, &m.Config)
```

Comment:
Is there a reason this changed?

Reply:
For me it was error'ing when I did `go build`. I didn't really intent for that to be part of this PR. I can switch it back if need be.

Reply:
If the parsing was failing, either the config is malformed or spin isn't tolerant enough. In either case it seems unrelated to this PR.

Reply:
Okay. I will update the PR.